manifests/init.pp

```diff
@@ -208,6 +208,9 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing. This requires Foreman 1.22+
 #
+# $rails_cache_store::            Set rails cache store. This requires Foreman 1.22+. Defaults to {'type' => 'file'}.
+#                                 For redis use {'type' => 'redis', 'urls' => ['localhost:8479/0'], options => {'compress' => 'true', 'namespace' => 'foreman'}}.
```

Comment:
In another issue we've had the problem that this is confusing for foreman-installer users. How about an example in README?

Reply:
I added a paragraph to the README. Was this the one you wanted or did you mean the one of foreman-installer?

Reply:
I think it can be dropped here now. You can also drop the 1.22+ part since the module is 2.0.0 only now.

Reply:
Could you have a look at this?

Reply:
Shortened to "Set rails cache store" only